docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -123,9 +128,128 @@ outbreaks due to lack of vaccination in small communities.
 Data Description
 ----------------
 
-.. todo::
+.. list-table:: Definitions
+   :widths: 25 50
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - S
+     - Susceptible to measles
+   * - I
+     - Infected with measles
+   * - R
+     - Recovered from measles
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_c341
+     - 
+   * - S
+     - excess mortality |br| rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c341
+     - 
+   * - I
+     - excess mortality |br| rate
+     - :math:`\frac{\text{deaths_c341}}{\text{population * prevalence_c341}}`
+     - 
+   * - I
+     - disability weights
+     - disability_weight_s117* |br| prevalence_s117+ |br| disability_weight_s118* |br| prevalence_s118
+     - GBD assumes 50% of measles |br| cases as severe and other 50% |br| as moderate |br| [GBD-2017-YLD-Capstone-Appendix-1]_.
+   * - R
+     - prevalence
+     - 0
+     - Clearly room for improvement. |br| This is done to simplify the model |br| as the focus is on LSFF but not |br| on measles.
+   * - R
+     - excess mortality |br| rate
+     - 0
+     - 
+   * - R
+     - disabilty weights
+     - 0
+     - 
+   * - ALL
+     - cause specific |br| mortality rate
+     - :math:`\frac{\text{deaths_c341}}{\text{population}}`
+     - 
+
+
+.. list-table:: Transition Data
+   :widths: 10 10 10 30 30
+   :header-rows: 1
+   
+   * - Transition
+     - Source 
+     - Sink 
+     - Value
+     - Notes
+   * - i
+     - S
+     - I
+     - incidence_rate_c341
+     - Incidence rate in |br| total population
+   * - r
+     - I
+     - R
+     - remission_rate_c341 |br| :math:`= \frac{\text{1}}{\text{10 person-days * 365 person-days}}` |br| :math:`= 36.5`
+     - GBD assumes average |br| case duration as 10 days |br| [GBD-2017-YLD-Capstone-Appendix-1]_. |br| So remission is approximated |br| to this calculation. 
+
+
+.. list-table:: Data Sources
+   :widths: 20 25 25 25
+   :header-rows: 1
+   
+   * - Measure
+     - Sources
+     - Description
+     - Notes
+   * - prevalence_c341
+     - COMO
+     - Prevalence of cause measles
+     - 
+   * - deaths_c341
+     - CoDCorrect
+     - Deaths from measles
+     - 
+   * - population
+     - Demography
+     - Mid-year population for |br| given country
+     - 
+   * - incidence_rate_c341
+     - COMO
+     - Incidence rate for measles
+     - 
+   * - remission_rate_c341
+     - Calculation
```

Comment:
Apologies if you've already discussed this, but I thought the calculation would appear in this table directly, not just the word "calculation"

Reply:
May be I misunderstood during the discussion, I have included calculation as value in the Transitions table. I will move that here.

Reply:
We just discussed putting YLD Appendix for the source, and adding a note in the Notes column that we are using a constant remission rate based on the average duration of 10 days that they assumed.